<ui version="4.0" >
 <class>Dialog_MaskROI</class>
 <widget class="QDialog" name="Dialog_MaskROI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>20</y>
     <width>332</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Mask Region :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="maskregionBox" >
        <item>
         <property name="text" >
          <string>XYZ cube by ROI intersection</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>XYZ cube by ROI union</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>XY Plane only</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>YZ Plane only</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>ZX Plane only</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" >
     <layout class="QHBoxLayout" name="horizontalLayout_6" >
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Mask Channels :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" >
     <layout class="QHBoxLayout" name="horizontalLayout_5" >
      <item>
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>first channel :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="firstchannelBox" />
      </item>
      <item>
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>last channel :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="lastchannelBox" />
      </item>
     </layout>
    </item>
    <item row="3" column="0" >
     <layout class="QHBoxLayout" name="horizontalLayout_4" >
      <item>
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>filling value :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="fillingvalueBox" />
      </item>
     </layout>
    </item>
    <item row="4" column="0" >
     <layout class="QHBoxLayout" name="horizontalLayout_3" >
      <item>
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>filling in or outside ROI :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fillingroiBox" >
        <item>
         <property name="text" >
          <string>Outside ROI</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Inside ROI</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="infolabel" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>280</y>
     <width>332</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>info: Mask ROI Options</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>224</y>
     <width>332</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" >
    <item>
     <spacer name="horizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton" >
      <property name="text" >
       <string>OK</string>
      </property>
      <property name="default" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
